app/ducks/walletReducer.js

```diff
@@ -40,6 +41,7 @@ export function getInitialState() {
     receiveDepth: 0,
     transactions: new Map(),
     idle: 0,
+    maxIdle: 5,
```

Comment:
i think this state should live in `app` duck instead of the `wallet` duck. wdyt?

Reply:
I placed it in wallet because the state `idle` and actions, reducers `resetIdle`, `incrementIdle` are all part of wallet already. But yeah it's more of a global thing, not wallet-specific. I'm fine with it either way.